plugins/inputs/http_response/http_response.go

```diff
@@ -239,6 +245,14 @@ func (h *HTTPResponse) httpGather(u string) (map[string]interface{}, map[string]
 	resp, err := h.client.Do(request)
 	response_time := time.Since(start).Seconds()
 
+	// Add the response headers
+	for headerName, measurementName := range h.HTTPHeaderTags {
```

Comment:
Rename `measurementName` -> `tagKey` or just `tag`.  This will reduce confusion against the metric measurement/name.

Reply:
@danielnelson done, please check again